### PR TITLE
En_Akindonuts

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

**Review by hensldm — COMMENTED** [review]
Couple thoughts.

File: src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c [hunk 1]
```diff
@@ -22,113 +64,1654 @@ const ActorInit En_Akindonuts_InitVars = {
...
+        if (!(gSaveContext.weekEventReg[63] & 0x8) && !(gSaveContext.weekEventReg[63] & 0x10)) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+s32 func_80BED208(EnAkindonuts* this) {
+    if (INV_CONTENT(ITEM_BEAN) != 10U) {
```
[the remark this answers is not in the document]
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c:305]: Using ITEM_MAGIC_BEANS doesn't work, and casting to u8 doesn't work either.

File: src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c [hunk 2]
```diff
@@ -28,113 +64,1654 @@ const ActorInit En_Akindonuts_InitVars = {
...
+    f32 sp40;
+    f32 sp3C;
+    Vec3f sp30;
+
+    Math_Vec3s_ToVec3f(&sp30, &sp5C[idx]);
+
+    if (idx == 0) {
+        phi_f12 = sp5C[1].x - sp5C[0].x;
+        phi_f14 = sp5C[1].z - sp5C[0].z;
+    } else if (((void)0, sp58) == (idx + 1)) {
```
[the remark this answers is not in the document]
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c:180]: Unfortunately not.

File: src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c [hunk 3]
```diff
@@ -28,113 +64,1654 @@ const ActorInit En_Akindonuts_InitVars = {
...
+            Audio_PlayActorSound2(&this->actor, NA_SE_EN_NUTS_DOWN);
+            func_8013BC6C(&this->skelAnime, sAnimations, 17);
+        } else if (this->unk_338 == 2) {
+            this->unk_338 = 16;
+            this->collider.dim.height = 32;
+            Audio_PlayActorSound2(&this->actor, NA_SE_EN_NUTS_UP);
+            func_8013BC6C(&this->skelAnime, sAnimations, 16);
+        } else if (this->unk_338 == 17) {
+            phi_v0 = DECR(this->unk_33A);
+            if (phi_v0 == 0) {
```
[the remark this answers is not in the document]
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Akindonuts/z_en_akindonuts.c:1204]: The second one works, but changing the first one gives a reglaloc v0/v1 flip.

**Review by AngheloAlf — APPROVED** [review]

File: undefined_syms.txt [hunk 4]
```diff
@@ -1539,10 +1539,27 @@ D_06009E70 = 0x06009E70;
...
+D_06003CC0 = 0x06003CC0;
+D_06004700 = 0x06004700;
+D_06004AA0 = 0x06004AA0;
+D_06004E38 = 0x06004E38;
 D_06005488 = 0x06005488;
+D_0600577C = 0x0600577C;
+D_06005CA8 = 0x06005CA8;
 D_06008290 = 0x06008290;
 D_0600AC70 = 0x0600AC70;
+D_0600B0B4 = 0x0600B0B4;
```
Comment (hensldm) [inline undefined_syms.txt:1562]: Should be able to remove these now right?

Comment (EllipticEllipsis) [reply undefined_syms.txt:1562]: And the ones from EnTru, which apparently I forgot to check.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**